code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/Message.kt

```diff
@@ -62,13 +69,50 @@ internal fun MessageScreen(
         onBackPressed()
     }
 
-    Message(
-        isVisible = presentationStateManager.visibilityState,
-        inAppMessageSettings = inAppMessageSettings,
-        gestureTracker = gestureTracker,
-        onCreated = { onCreated(it) },
-        onDisposed = { onDisposed() }
-    )
+    if (inAppMessageSettings.shouldTakeOverUi) {
+        /**
+         * Dialog is used to take over the UI when the InAppMessage is set to take over the UI.
+         * This is necessary to ensure that the InAppMessage is displayed on top of the UI.
+         * Which will ensure that ScreenReader can read the content of the InAppMessage only and not the underlying UI.
+         */
+        Dialog(
+            properties = DialogProperties(
+                usePlatformDefaultWidth = false,
+                dismissOnBackPress = true,
+                dismissOnClickOutside = false
+            ),
+            onDismissRequest = {
+                onBackPressed()
+            }
+        ) {
+            /** Remove the default dim and animations for the dialog window
+             * Customer can set their own dim and animations if needed and those will be honoured in MessageBackdrop inside Message
+             */
```

Comment:
Please switch this into a multi-line comment instead of a doc comment.

Reply:
Done